bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx90-en.html

```diff
@@ -0,0 +1,129 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "firefox/whatsnew/base.html" %}
+
+{% block page_title %}What’s new with Firefox{% endblock %}
+
+{#- This will appear as <meta property="og:description"> which can be used for social share -#}
+{% block page_og_desc %}What’s new with Firefox{% endblock %}
+
+{% block body_id %}firefox-whatsnew{% endblock %}
+{% block body_class %}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('firefox_whatsnew_90') }}
+{% endblock %}
+
+{% block experiments %}
+  {% if switch('firefox-whatsnew90-experiment', ['en-US', 'en-CA']) %}
+    {{ js_bundle('firefox_whatsnew90_experiment') }}
+  {% endif %}
+{% endblock %}
+
+{% set _entrypoint = 'mozilla.org-whatsnew90' %}
+
+{% set utm_campaign = 'wnp-90' %}
```

Comment:
Should this be `wnp90` without the hyphen?  (just a difference I noticed to the links Dan shared, but it probably matters very little).